### ISSUE TITLE
It seems that the download websites are bloken.

### ISSUE THREAD

**Issue by YaqiWangCV** [ofsoundof/LSDIR#6]

**Comment by MaximeSongIdris** [comment]
Yes, it is the same for me too, the download website isn't accessible anymore

**Comment by SoHardToMakeAName** [comment]
Yes, please recover it :)

**Comment by djp0246** [comment]
Hello, I urgently need LSDIR this dataset now, could you fix the download website,  or is there any kind person who can share this dataset, thank you very much.
